product_warranty/__openerp__.py

```diff
@@ -22,23 +22,23 @@
 
 {
     'name': 'Product warranty',
-    'version': '8.0.1.0.0',
+    'version': '9.0.1.0.0',
     'category': 'Generic Modules/Product',
     'author': "Akretion,Odoo Community Association (OCA),Vauxoo",
     'website': 'http://akretion.com',
     'license': 'AGPL-3',
     'depends': ['product'],
     'data': [
         'security/ir.model.access.csv',
-        'views/res_company_view.xml',
-        'views/product_warranty_view.xml',
+        'views/res_company.xml',
```

Comment:
Don't rename these files. They are good with the `_view` suffix. You can have also a `res_company_data.xml` file for example for insert data in the companies, and thus the name will be ambiguous without the suffix.